src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const promise1 = new Promise((resolve, reject) => {
+    document.querySelector('.logo').addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  });
+
+  const promise2 = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      reject('Promise was rejected!');
+    }, 3000);
+  });
+
+  promise1
+    .then((message) => {
+      const successDiv = document.createElement('div');
+
+      successDiv.className = 'message';
+      successDiv.innerText = message;
+      document.body.appendChild(successDiv);
```

Comment:
better to create some common helper function and just call it in every then and catch